src/pages/alumni/ProfilePage.tsx

```
@@ -153,13 +159,18 @@ const AlumniProfileContent = ({
     },
   });
 
+  // 본인 프로필 여부 (본인 프로필에서는 신고 옵션 자체를 노출하지 않음)
+  const isMine = String(loginUserId ?? '') === String(profile.userId);
+
   // 다른 프로필로 이동 시 로컬 상태를 초기화합니다.
   useEffect(() => {
     setIsFollowing(profile.isFollowing);
```

ESLint

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/src/pages/alumni/ProfilePage.tsx:167:5
  165 |   // 다른 프로필로 이동 시 로컬 상태를 초기화합니다.
  166 |   useEffect(() => {
> 167 |     setIsFollowing(profile.isFollowing);
      |     ^^^^^^^^^^^^^^ Avoid calling setState() directly within an effect
  168 |     setFollowerCount(profile.followerCount);
  169 |     setIsFollowPending(false);
  170 |     setPopUpConfig(null);

(react-hooks/set-state-in-effect)